src/odemis/driver/simulated.py

```diff
@@ -339,3 +238,195 @@ def __init__(self, name, role, positions, has_pressure=False, **kwargs):
         # TODO: set to None/None when the sample is ejected
         self.sampleHolder = model.TupleVA((PHENOM_SH_FAKE_ID, PHENOM_SH_TYPE_OPTICAL),
                                          readonly=True)
+
+
+class GenericComponent(model.Actuator):
+    """
+    Simulated component, capable of simulating both axes and VA's.
+    This component allows for communication with it, asif it were a real component with axes and VA's, but does not do
+    anything else.
+    """
+    def __init__(self, name, role, vas=None, axes=None, **kwargs):
```

Comment:
I don't get when/why you want to use this generic component. I also don't get why it is not based on HwComponent instead of Actuator. In principle the component is now a generic sub component of Actuator. Suggesting it is more a "generic actuator"?

Reply:
Please add a small comment above to explain this. 
BTW, an option would be to have a GenericComponent(HwComponent) and a subclass GenericActuator(GenericComponent, Actuator), fix the possible diamond inheritance issues in the model, and use a __new__() method in GenericComponent() to generate a GenericActuator if there are axes... but that sounds a little over the top for such small class.